app/frontend/styles/_notes.scss

```diff
@@ -9,6 +9,8 @@
   padding: 1px 0px 1px 15px;
   margin-top: 10px;
   margin-bottom: 5px;
+  width: fit-content;
```

Comment:
Seems incompatible with IE11: https://caniuse.com/#feat=mdn-css_properties_width_fit-content

Reply:
Service manual still recommends "compliant" in IE11: https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices

Reply:
I think this is quite a small thing, but I'm going to take a look locally to see if there's an easy fix which doesn't use this property